.github/workflows/pr_ci.yml

```diff
@@ -15,17 +15,15 @@ jobs:
     name: Build & Test
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-node@v2
+    - uses: actions/cache@v3
       with:
-        node-version: "15"
-    - name: Setup .NET 6.0
-      uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: 6.0.x
-        include-prerelease: true
+         path: ~/.nuget/packages
+         key: ${{ runner.os }}-nuget-${{ hashFiles('**/packages.lock.json') }}
+         restore-keys: |
+            ${{ runner.os }}-nuget-
     - name: Build
       run: |
-        dotnet build --configuration Release --framework net6.0
+        dotnet build --configuration Release --framework net6.0 /p:CreateHardLinksForCopyLocalIfPossible=true /p:CreateHardLinksForCopyFilesToOutputDirectoryIfPossible=true /p:CreateHardLinksForPublishFilesIfPossible=true
```

Comment:
I tried this out with the OSOCE solution, see the results here: https://github.com/Lombiq/GitHub-Actions/issues/64#issuecomment-1275348134 No luck with performance.

Reply:
can try them here too, I think these mostly should help with spinning disks, not sure what's the runner like in GHA

Reply:
It's an old D2-tier Azure VM, which is supposed to use HDDs (I tested performance on GitHub Actions, not my own machine or else).